flocker/common/functional/test_era.py

```diff
@@ -0,0 +1,31 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Functional tests for ``flocker-node-era`` command.
+"""
+
+from unittest import skipUnless
+from subprocess import check_output
+
+from twisted.python.procutils import which
+
+from .._era import get_era
+from ...testtools import make_script_tests
+
+EXECUTABLE = b"flocker-node-era"
+
+
+class FlockerNodeEraTests(make_script_tests(EXECUTABLE)):
+    """
+    Tests for ``flocker-node-era``.
+    """
+    @skipUnless(which(EXECUTABLE), EXECUTABLE + " not installed")
```

Comment:
We're installing `flocker-node-era` on non-Linux platforms where it is near guaranteed not to work, e.g. http://build.clusterhq.com/builders/flocker-osx-10.10/builds/6116/steps/trial/logs/problems

We should either:
1. Not install it
2. Have it fail gracefully with a 'not supported on this platform' error message

2. feels cleaner to me.